doc/source/tune-usage.rst

```diff
@@ -459,7 +459,7 @@ You often will want to compute a large object (e.g., training data, model weight
 
     import ray
     from ray import tune
-    from ray.tune.util import pin_in_object_store, get_pinned_object
+    from ray.tune.utils.util import pin_in_object_store, get_pinned_object
```

Comment:
```suggestion
    from ray.tune.utils import pin_in_object_store, get_pinned_object
```

Reply:
A yes, correct. It is imported in the __init__.py